dll/win32/uxtheme/buffer.c

```diff
@@ -19,6 +19,29 @@
  */
 
 #include "uxthemep.h"
+#include <wine/heap.h>
+
+#if (DLL_EXPORT_VERSION >= _WIN32_WINNT_VISTA)
+struct paintbuffer
+{
+    HDC targetdc;
+    HDC memorydc;
+    HBITMAP bitmap;
+    RECT rect;
+    void *bits;
+};
+
+static void free_paintbuffer(struct paintbuffer *buffer)
+{
+    DeleteObject(buffer->bitmap);
+    DeleteDC(buffer->memorydc);
+    heap_free(buffer);
+}
+
+static struct paintbuffer *get_buffer_obj(HPAINTBUFFER handle)
+{
+    return handle;
```

Comment:
lol they need a full blown function just to do a cast?

Reply:
...is this supposed to be actionable feedback for this PR?